README.md

```diff
@@ -568,8 +568,19 @@ data = MarketData.UpsertData(mkdid, 'CET',
   )
 
 mkservice.upsertData(data)
+```
 
+Upsert has optional switches that can be applied
+```
+mkservice.upsertData(data, deferCommandExecution, deferDataGeneration, keepNulls, upsertMode)
 ```
+The switch details are,
+
+deferCommandExecution (true/false) choose between syncronoys and asyncronous command execution, default is false.
+deferDataGeneration (true/false) choose between syncronoys and asyncronous precomputed data generation, default is true.
+keepNulls (true/false) if true then nulls are written in the curve replacing any data present for the instant, default is false.
+upsertMode (Merge/Replace) for ActualTimeSeries the two modes are equivalent, default is merge.
```

Comment:
in the UpsertData class, the default is None, not Merge. and None doesn't explain what None translate to.
If is Merge, the DTO should default to Merge or a more articulate description is needed.

Reply:
Is it better to have upsertmode in the UpsertData class as, `upsertMode: UpsertMode = UpsertMode.Merge` ?

I currently have `upsertMode: Optional[UpsertMode] = None`, that effectively sends null to the service and there null is the same as Merge. So, I figure I'm conflating what's in the SDK and the service.